packages/dnb-eufemia/src/components/tabs/style/dnb-tabs.scss

```
@@ -4,8 +4,22 @@
  */
 
 @import '../../../style/core/utilities.scss';
+@import '../../button/style/themes/button-mixins.scss';
 
 .dnb-tabs {
+  --tablist-border-color--default: grey;
+  --tab-title-font-weight: var(--font-weight-regular);
+  --tab-title-font-size: var(--font-size-basis);
+  --tab-title-font-color: #0c1115;
```

```suggestion
  --tab-title-font-color: DimGray;
```

Its just a dummy fallback, right? Maybe we can give it then a CSS name instead.